tests/foreman/ui/test_rhcloud_insights_vulnerability.py

```diff
@@ -0,0 +1,80 @@
+"""Tests for RH Cloud - Vulnerability
+
+:Requirement: RHCloud
+
+:CaseAutomation: Automated
+
+:CaseComponent: RHCloud
+
+:Team: Proton
+
+:CaseImportance: High
+
+"""
+
+import pytest
+
+
+def create_insights_vulnerability(host):
+    """Function to create vulnerabilities for IoP Vulnerability Engine"""
+    # Upload insights data to Satellite
+    assert host.execute('insights-client').status == 0
+
+    # Add vulnerability for glibc package - CVE-2025-8058
+    assert host.execute('dnf downgrade -y glibc').status == 0
+
+
+@pytest.mark.e2e
+@pytest.mark.no_containers
+@pytest.mark.rhel_ver_list([10])
+@pytest.mark.parametrize('module_target_sat_insights', [False], ids=['local'], indirect=True)
+def test_rhcloud_insights_vulnerabilities_e2e(
+    request,
+    rhel_insights_vm,
+    rhcloud_manifest_org,
+    module_target_sat_insights,
+):
+    """Validate vulnerability data from local Insights,
+        verify results are displayed correctly for impacted host in Satellite.
+
+    :id: d952e83c-3faf-4299-a048-2eb6ccb8c9c3
+
+    :steps:
+        1. Enable local Insights/Red Hat Lightspeed on Satellite.
+        2. Import Manifest, Sync RHEL content and create AK with the content.
+        3. Register a VM using AK created.
+        4. Create vulnerabilities for detection by vulnerability engine and upload its data to Insights.
+        5. In Satellite UI, go to vulnerabilities tab on host details page and validate the data.
+
+    :expectedresults:
+        1. Local insights vulnerability engine detects the new vulnerabilities.
+        2. Vulnerability data is displayed correctly for impacted host in Satellite.
+
+    :verifies: SAT-30762
+    """
+    org_name = rhcloud_manifest_org.name
+    # local_advisor_enabled = module_target_sat_insights.local_advisor_enabled
+
+    # Verify insights-client can update to latest version available from server
+    assert rhel_insights_vm.execute('insights-client --version').status == 0
+
+    # Manual VMaaS reposcan sync task
+    assert (
+        module_target_sat_insights.execute(
+            'curl --cert /etc/foreman/client_cert.pem --key /etc/foreman/client_key.pem --cacert /etc/foreman/proxy_ca.pem -X PUT "https://localhost:24443/api/vmaas-reposcan/v1/sync"'
```

Comment:
```suggestion
            'curl --fail --cert /etc/foreman/client_cert.pem --key /etc/foreman/client_key.pem --cacert /etc/foreman/proxy_ca.pem -X PUT "https://localhost:24443/api/vmaas-reposcan/v1/sync"'
```
If you want to make sure the command succeeds, you need to add the `--fail` flag

Reply:
@vkrizan is there a way to know when the sync is done? I know that the action is async